src/controllers/playback/video/index.js

```diff
@@ -97,9 +101,9 @@ import 'css!assets/css/videoosd';
                 return void view.querySelector('.btnRecord').classList.add('hide');
             }
 
-            window.connectionManager.getApiClient(item.ServerId).getCurrentUser().then(function (user) {
+            ServerConnections.getApiClient(item.ServerId).getCurrentUser().then(function (user) {
                 if (user.Policy.EnableLiveTvManagement) {
-                    import('recordingButton').then(({default: RecordingButton}) => {
+                    import('../../../components/recordingcreator/recordingbutton').then((RecordingButton) => {
```

Comment:
would be better to just require this at the beginning of the file, but this will load every css on the start of the app. We could use the lazyStyleTag injectType from style-loader. Then you could call use() or unuse() from the imported css so it could be injected on demand on the dom. Something to consider.